templates/engage/index.html

```diff
@@ -19,56 +19,101 @@
     <div class="col-6">
       <h1>Engage pages</h1>
       <p>Ubuntu case studies, whitepapers and webinars.</p>
-      <a class="p-button--positive p-link--external" href="https://discourse.ubuntu.com/t/engage-pages-and-takeovers/17229">Add new engage pages</a>
-      <a class="p-button--neutral p-link--external" href="https://discourse.ubuntu.com/t/how-to-add-engage-pages/18185">How to add engage pages</a>
     </div>
   </div>
 </section>
 
-
 <section class="p-strip is-shallow">
+  <form action="/">
   <div class="row">
-    <div class="col-4">
+    <div class="col-3">
       <div class="p-form__group">
-        <form action="/">
-          <label for="language-selector">Select language:</label>
-          <select name="language-selector" id="language-selector" class="u-no-margin--bottom">
-            <option value="all">All languages</option>
-            <option value="de">Deutsch</option>
-            <option value="en">English</option>
-            <option value="es">Espa&nacute;ol</option>
-            <option value="fr">Fran&ccedil;ais</option>
-            <option value="it">Italiano</option>
-            <option value="pt">Portugu&ecirc;s</option>
-          </select>
-        </form>
+        <label for="language-selector">Select language:</label>
+        <select name="language-selector" id="language-selector" class="u-no-margin--bottom">
+          <option value="all">All languages</option>
+          <option value="de">Deutsch</option>
+          <option value="en">English</option>
+          <option value="es">Espa&nacute;ol</option>
+          <option value="fr">Fran&ccedil;ais</option>
+          <option value="it">Italiano</option>
+          <option value="pt">Portugu&ecirc;s</option>
+        </select>
       </div>
     </div>
+    {% if resource_types %}
+    <div class="col-3">
+      <div class="p-form__group">
+        <label for="resource-selector">Select resource type:</label>
+        <select name="resource-selector" id="resource-selector" class="u-no-margin--bottom">
+          <option value="all">All resource types</option>
+          {% for resource in resource_types %}
+            <option value="{{ resource }}">{{ resource | capitalize }}</option>
+          {% endfor %}
+        </select>
+      </div>
+    </div>
+    {% endif %}
+    {% if tags %}
+    <div class="col-3">
+      <div class="p-form__group">
+        <label for="tag-selector">Select tag:</label>
+        <select name="tag-selector" id="tag-selector" class="u-no-margin--bottom">
+          <option value="all">All tags</option>
+          {% for tag in tags %}
+            <option value="{{ tag }}">{{ tag }}</option>
+          {% endfor %}
+        </select>
+      </div>
+    </div>
+    {% endif %}
+    <div class="col-3" style="display: flex; align-items: flex-end; justify-content: flex-end;">
+      <button id="clear-filters" class="u-no-margin--bottom" onclick="clearFilters()" type="reset" disabled>Clear filters</button>
```

Comment:
Could you add a `<br class="u-hide u-show--small" />` above this button to add some space on small screens:
![ubuntu-com-10192 demos haus_engage_language=en resource=whitepaper tag=maas](https://user-images.githubusercontent.com/1413534/129712306-b0f9f077-883c-4a7d-817e-78556601b1c4.png)